.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb',
```

Comment:
You may want to add this so Eslint can recognize React code:
`  extends: [
    'plugin:react/recommended',
    'airbnb',
  ],`

Reply:
I have...
```
plugins: [
    'react',
  ],
``` 
Is this the same thing?

Reply:
I'm not sure. I must have copied that from a Stackoverflow post and I can't find the reasoning when I searched again. If it works without it, then I guess it's not necessary.

Reply:
I'll see if it works. Thanks, Andrew!

Reply:
I followed this to set up the eslint config https://eslint.org/docs/6.0.0/user-guide/getting-started

this command specifically ./node_modules/.bin/eslint --init

Reply:
Yes, I'll see if it works. Thanks, Andrew!

Reply:
I followed this one (similar) https://eslint.org/docs/user-guide/getting-started. :). I run `npx eslint --init` instead of `./node_modules/.bin/eslint --init`